### PR TITLE
fix the scope of some rules with dependencies

### PR DESCRIPTION
**Pull request by williballenthin**
ref: https://github.com/mandiant/capa/issues/2124

### CODE REVIEW

File: linking/runtime-linking/link-function-at-runtime-on-windows.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ rule:
       - moritz.raabe@mandiant.com
       - michael.hunhoff@mandiant.com
     scopes:
-      static: function
-      dynamic: unsupported  # requires characteristic features
+      static: instruction
+      dynamic: call
```
Comment (mr-tz) [inline linking/runtime-linking/link-function-at-runtime-on-windows.yml:10]: with the optional part we're targeting multiple API calls which won't work this way

Comment (williballenthin) [reply linking/runtime-linking/link-function-at-runtime-on-windows.yml:10]: ah, right...

I'm inclined to remove the `optional` and let this be used as a dependency more widely. Thoughts?

Comment (williballenthin) [reply linking/runtime-linking/link-function-at-runtime-on-windows.yml:10]: or relax scope to `basic block`

**Review by mike-hunhoff — COMMENTED** [review]
Looks good pending @mr-tz comments, thanks!

File: linking/runtime-linking/link-function-at-runtime-on-windows.yml (hunk 1 above)
Comment (mr-tz) [reply linking/runtime-linking/link-function-at-runtime-on-windows.yml:10]: removed in 0c7d1bdfacac57cc6480b1e9f2e60019c1a130c7

**Pull request merged**